signIn.html

```diff
@@ -83,7 +83,7 @@ <h1 class="heading"><span>SCALE</span>TECH</h1>
            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR7H0u13wFrskVIje5pLMN3kCExmMAQzYOKaQwMYiHNT03LUJIiaoCkTBPqjwRpund-0hU&usqp=CAU" alt="scaletech logo" height="400px" width="400px">
         </div>
         <div class="details">
-            <form action="#" method="POST" >
+            <form action="" method="POST" onsubmit="sendData()">

```

Comment:
Here you must pass the event explicitly to handle that in javascript code, and the handler code should return a boolean for validation.

onsubmit="return sendData(event)"

Reply:
👍